server/sonar-server/src/main/java/org/sonar/server/component/ws/TreeAction.java

```diff
@@ -55,10 +55,10 @@
 import org.sonarqube.ws.client.component.TreeWsRequest;
 
 import static com.google.common.base.Preconditions.checkArgument;
-import static com.google.common.collect.FluentIterable.from;
 import static java.lang.String.CASE_INSENSITIVE_ORDER;
 import static java.lang.String.format;
 import static java.util.Collections.emptyMap;
+import static java.util.stream.Collectors.toList;
```

Comment:
In fact we're using our own Collectors (returning immutable lists) => please use MoreCollectors instead

Reply:
done